.travis.yml

```diff
@@ -0,0 +1,31 @@
+sudo: required
+
+language: scala
+
+services:
+- docker
+
+cache:
+  directories:
+  - $HOME/.ivy2/cache
+  - $HOME/.sbt/boot/
+
+script:
+  - /bin/true
```

Comment:
Unfortunately we don't have tests, and I'm not sure how easy / difficult that would be since everything is so tied to Spark, and we don't have enough isolation to test things by themselves.

But we could just do `./sbt "project server" compile` which doesn't take too long and doesn't assemble all dependencies:

```bash
$ time ./sbt "project server" compile
[info] Loading project definition from /Users/ttuhinanshu/dev/ufmg/project
[info] Set current project to usaceflood (in build file:/Users/ttuhinanshu/dev/ufmg/)
[info] Set current project to usaceflood-server (in build file:/Users/ttuhinanshu/dev/ufmg/)
[success] Total time: 0 s, completed Jan 18, 2016 11:31:00 AM

real   0m5.608s
user   0m14.614s
sys    0m0.730s
```